dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -725,28 +725,32 @@ public VersionTypeValidation validateWorkflowSet(Set<SourceFile> sourcefiles, St
         Optional<SourceFile> mainDescriptor = filteredSourcefiles.stream().filter((sourceFile -> Objects.equals(sourceFile.getPath(), primaryDescriptorFilePath))).findFirst();
 
         boolean isValid = true;
-        String validationMessage = null;
+        StringBuilder validationMessage = new StringBuilder();
         Map<String, String> validationMessageObject = new HashMap<>();
 
         if (mainDescriptor.isPresent()) {
             Yaml yaml = new Yaml();
             String content = mainDescriptor.get().getContent();
             if (content == null || content.isEmpty()) {
                 isValid = false;
-                validationMessage = "Primary descriptor is empty.";
+                validationMessage.append("Primary descriptor is empty.");
             } else if (!content.contains("class: Workflow")) {
                 isValid = false;
-                validationMessage = "Requires class: Workflow.";
+                validationMessage.append("A CWL workflow requires 'class: Workflow'.");
```

Comment:
I would personally use double quotes, but I can go either way: `"...\"class: Workflow\"."`

Reply:
This is what it would look like with double quotes:
![Screenshot from 2020-02-07 13-17-02](https://user-images.githubusercontent.com/25287123/74054515-345e0180-49ac-11ea-943b-e3aa664b7eb5.png)

It also looks like single quotes are used for some error messages, such as the one for /Dockstore.wdl when the file has no content. Do you think it would be better to just keep it as single quotes for uniformity?

What it looks like with single quotes:
![Screenshot from 2020-02-07 13-25-12](https://user-images.githubusercontent.com/25287123/74055091-586e1280-49ad-11ea-8cae-74efb4638ea4.png)

Reply:
Ah, sorry, I did not look at other messages for comparison before I wrote my comment... If we're already using single quotes elsewhere, especially in the same message, we should be consistent and continue to use single quotes.

I was commenting from more of a grammar perspective, although I don't actually know what the official rule is on that -- it just felt right to use double quotes. :)